### ISSUE TITLE
"Button" title without photos in the screen

### ISSUE THREAD

**Issue by seriyvolk83** [mikaoj/BSImagePicker#67]
When I first time invoke this picker I see empty screen with "Button" title. After that if I again open the picker it's ok.
I think the problem is related to the asked permutations.

**Comment by sherifnada** [comment]
+1 I am also having this issue.

**Comment by mikaoj** [comment]
I'll look into it, thanks for reporting it!

A quickfix for now could be for you to ask for permissions to the photo library yourself before presenting the picker.

**Comment by sherifnada** [comment]
Temporary workaround to ask for permissions: 

```
PHPhotoLibrary.requestAuthorization { status in
    switch status {
    case .Authorized:
        <#your code#>
    case .Restricted:
        <#your code#>
    case .Denied:
        <#your code#>
    default:
        // place for .NotDetermined - in this callback status is already determined so should never get here
        break
    }
}
```
[Source](http://stackoverflow.com/a/26656177)

**Comment by mikaoj** [comment]
Fixed in 2.3.0
